app/models/edition/role_appointments.rb

```diff
@@ -21,4 +21,11 @@ def can_be_associated_with_role_appointments?
   def search_index
     super.merge("people" => role_appointments.map(&:slug))
   end
+
+  def is_associated_with_a_minister?
```

Comment:
This is just copying the approach you took in https://github.com/alphagov/whitehall/commit/4ead19a72fc28c604edb8b4c206b22117accf567 which doesn't check super on any of the other instances of this method. If we make that change here then the others should also be updated.